extensions/.import-restrictions

```diff
@@ -2,6 +2,7 @@ rules:
 - selectorRegexp: github[.]com/gardener/gardener
   # avoid pulling in the entire gardener packages, most of which should be of no interest to extensions
   allowedPrefixes:
+  - github.com/gardener/gardener/pkg/operation/botanist/component
```

Comment:
make verify was complaining that `github.com/gardener/gardener/pkg/operation/botanist/component` should be added in the `allowedPrefixes`.

Reply:
It should be because there are components under the extension package: example: 
 `pkg/operation/botanist/component/extensions/operatingsystemconfig/downloader/logging.go`
that import `github.com/gardener/gardener/pkg/operation/botanist/component`

Reply:
I have troubles understanding this - could you perhaps add links to the files? As said before, this import restriction exists on purpose, and we should avoid lifting them if there are reasonable ways to do so.